src/unit.rs

```diff
@@ -2188,7 +2188,8 @@ impl<'input, 'abbrev, 'unit, Endian> EntriesCursor<'input, 'abbrev, 'unit, Endia
                 // down one level.
                 depth += 1;
 
-                let sibling_ptr = try!(self.current().unwrap().attr_value(constants::DW_AT_sibling));
+                let sibling_ptr =
+                    try!(self.current().unwrap().attr_value(constants::DW_AT_sibling));
```

Comment:
Yeah, but I don't feel like it should be a part of this PR. In the larger context it probably doesn't matter, but this change could probably be pushed separately (maybe even without a PR)